### PR TITLE
fix(event-chart): Display human readable time

### LINKED ISSUE #1674: Event chart shows unreadable time format
# Bug Report

## Expected Behavior

Users should understand time labels easily, like '1 min 15s'

## Current Behavior

Time axis shows '1.25 min'

## Possible Solution

Use different params for the format pipe

### PR DESCRIPTION
**Pull request by subarroca**
### <strong>Pull Request</strong>
Show human readable time format if units are bigger than seconds
Closes #1674 

#### Type of PR
Bugfix (non-breaking change which fixes an issue)

#### Checklist

- [x] I have read the CONTRIBUTING doc and I follow the PR guidelines
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: libs/barista-components/event-chart/src/event-chart.spec.ts [hunk 1]
```diff
@@ -164,6 +164,8 @@ describe('DtEventChart', () => {
         color: 'default',
       });
 
+      console.dir(renderedEvents);
```
Comment (rowa-audil) [inline libs/barista-components/event-chart/src/event-chart.spec.ts:167]: Why do you need to log this?

**Review by rowa-audil — COMMENTED** [review]
NIT otherwise LGTM! 🦝

**Review by lukasholzer — DISMISSED** [review]
Cool Thx 👍

**Review by rowa-audil — APPROVED** [review]

**Comment by rowa-audil** [comment]
LGTM! 🐧

**Review by thomaspink — APPROVED** [review]
LGTM. Thank you :)

File: apps/dev/src/event-chart/big-time-test-data.ts [hunk 2]
```diff
@@ -0,0 +1,224 @@
...
+
+export class BigTimeDataSource implements EventChartDemoDataSource {
+  getEvents(): EventChartDemoEvent[] {
+    const events: EventChartDemoEvent[] = [];
+    BIG_TIME_TEST_DATA.forEach((event) => {
+      let color: DtEventChartColors = 'default';
+      if (event.source.apdexRating !== 'Satisfying') {
+        color = 'error';
+      }
+      // if (event.source.name.includes('WithWebRequest')) {
```
Comment (bardosmisi) [inline apps/dev/src/event-chart/big-time-test-data.ts:163]: Commented out code should be removed.

**Comment by rowa-audil** [comment]
@subarroca Please don't squash and force push until the final approval as it makes reviewing the changes harder 🙂

**Review by rowa-audil — APPROVED** [review]

**Review by thomaspink — APPROVED** [review]

**Review by bardosmisi — APPROVED** [review]

**Review by peter-affenzeller — APPROVED** [review]

**Pull request merged**